pkgs/by-name/cr/cryptpad/package.nix

```diff
@@ -121,7 +130,11 @@ buildNpmPackage {
     mkdir -p "$out_cryptpad/www/common/onlyoffice/dist"
     ${lib.concatMapStringsSep "\n" onlyoffice_install onlyoffice_versions}
     ${x2t_install}
-    rdfind -makehardlinks true -makeresultsfile false "$out_cryptpad/www/common/onlyoffice/dist"
```

Comment:
Ah, good point. I've updated my patch to try to make this clearer: it's now called `--check` rather than `--check-only`, and I've added `--rdfind` and `--no-rdfind` options to make it obvious that we're doing a rdfind here. A side benefit of this is that if we somehow forgot to install `rdfind`, we'll fail to build, rather than silently stop running rdfind.